server/src/test/java/com/broadcom/lsp/cobol/service/DocumentExtensionTests.java

```diff
@@ -34,7 +35,7 @@
  * extensions. If an extension of a document is unsupported (i.e. not "cob", "cbl" or "cobol"
  * ignoring case) then the according to notification should be sent.
  */
-class DocumentExtensionTests {
+class DocumentExtensionTests extends ConfigurableTest {
```

Comment:
In this case, we will create a connection to DI that we are actually want to get rid of. Will work for now; may be fixed in #261